src/lib.rs

```diff
@@ -108,6 +115,64 @@ pub fn perform_cracking(text: &str, config: Config) -> Option<DecoderResult> {
     config::set_global_config(modified_config);
     let text = text.to_string();
 
+    /* Initializing database */
+    let db_result = storage::database::setup_database();
+    match db_result {
+        Ok(_) => (),
+        Err(e) => {
+            cli_pretty_printing::warning(&format!(
+                "DEBUG: lib.rs - SQLite database failed to initialize. Encountered error: {}",
+                e
+            ));
+        }
+    };
```

Comment:
i presume in the future during our initial setup we'll offer the user the ability to download an already cached db, so this warning is good